doc/tutorials/plugins.md

```diff
@@ -441,6 +441,24 @@ Plugin *ELEKTRA_PLUGIN_EXPORT
 
 For further information see [the API documentation](https://doc.libelektra.org/api/current/html/group__plugin.html).
 
+### `elektraPluginGetGlobalKeySet`
+
+In order to enable communication between plugins which is more complex than what can be done with metadata, Elektra provides a global keyset which plugins can read from and modify.
+
+The keyset is initialized by the KDB and can be accessed by all plugins except for plugins created manually (e.g. with `elektraPluginOpen`).
+
+It can be accessed by calling the following function:
+
+```c
+KeySet * elektraPluginGetGlobalKeySet (Plugin * plugin);
+```
+
+The handle of the plugin using the keyset needs to be provided as `plugin`.
```

Comment:
I don't think this has to be mentioned (not here). Passing this `Plugin *` is used in all exported functions of plugins and lots of other places. If this info is missing, maybe mention it somewhere in a more general section.